EmployeeSort/Employee.rb

```diff
@@ -0,0 +1,32 @@
+require "test/unit"
+
+class Employee
+include Comparable
+attr_accessor :name
+
+  def <=>nextEmployeeObject
```

Comment:
Please add braces for argument declaration.  The argument name is not following the naming convention of snake_case. The name it self could be a better one.